rero_ils/templates/rero_ils/head.html

```diff
@@ -63,5 +63,6 @@
   {%- block header %}{% endblock header %}
   {%- block css %}
     {% assets "rero_ils_main_css" %}<link href="{{ ASSET_URL }}" rel="stylesheet">{% endassets %}
+    <link href="{{ config.get('RERO_ILS_THEME_ORGANISATION_CSS_ENDPOINT') }}{{ viewcode }}.css" rel="stylesheet" title="{{ viewcode }}">
```

Comment:
Can you check that this works in production (non debug mode). Probably it will failed due to APP_DEFAULT_SECURE_HEADERS policies.

Reply:
You're right this approach is the simplest I found regarding given constraint: change the CSS even if the server is launched. So yes I know that's a naive one. And that in the future it will evolve.

It avoids problems with static files that are (re)generated at each deployment, that the production don't keep them in kubernetes and so on.

I also think about `APP_DEFAULT_SECURE_HEADERS` policy, and found nothing about CSS and images.
Seems to be OK with `ENV=production` in **invenio.cfg** file.